test/functional/placeholders.js

```diff
@@ -84,12 +84,12 @@ describe('placeholders', () => {
 
     it('should warn about a missing key', () => {
       var fn = () => patch(container, render, {});
-      expect(fn).to.throw('Placeholder elements must have a key specified.');
+      expect(fn).to.throw(Error);
```

Comment:
I think asserting on the error message is nice because that way we can make sure the correct error was thrown.